opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -250,7 +252,7 @@ def get_default(cls) -> "TraceOptions":
 DEFAULT_TRACE_OPTIONS = TraceOptions.get_default()
 
 
-class TraceState(typing.Dict[str, str]):
+class TraceState(OrderedDict):
```

Comment:
I'm not happy about that but it seems the best we can do since there is no generic OrderedDict. (And all dicts being ordered is only guaranteed since 3.7 and implemented since 3.6).

Reply:
there's no need for this anymore since we're making the "TraceState" object in otel more generic.